howto/install-appliance/azure.md

```diff
@@ -0,0 +1,99 @@
+  [note type="caution" status="Warning"]The *Ubuntu Advantage for **Infrastructure*** token that every user gets for free for personal use does **NOT** work and will result in a failed deployment. You must purchase a *Ubuntu Advantage for **Applications*** subscription by [contacting Canonical](https://anbox-cloud.io/contact-us).[/note]
+* An Azure account that you use to create the virtual machine.
+
+## Create a virtual machine
+
+Complete the following steps to create a virtual machine on which you can install the Anbox Cloud Appliance.
+
+### 1. Deploy a Linux virtual machine
+
+Log on to the [Microsoft Azure Portal](https://portal.azure.com/) and select the **Quickstart Center** service.
+
+![Quickstart Center](../../images/azure_quickstart-co.png)
+
+In the Quickstart Center, select **Deploy a virtual machine**. On the resulting screen, select **Create a Linux virtual machine**.
+
+![Deploy a virtual machine](../../images/azure_deploy-vm-co.png)
+
+
+### 2. Configure basic settings
+
+On the **Basics** tab of the virtual machine configuration, specify the required information. Several of the options are specific to how and where you want to deploy your virtual machine. In most cases you can keep the default values, but make sure to set the following configurations:
+
+* Select the latest Ubuntu image (Ubuntu Server 22.04 LTS) for the architecture that you want to use. The following instructions and screenshots use the Arm64 architecture.
+* Select a size that matches the [hardware requirements](https://discourse.ubuntu.com/t/requirements/17734#appliance). For example, select `Standard_D16ps_v5`, which has 16 vCPUs and 64 GB of RAM.
+* Change the user name of the administrator account to `ubuntu`.
+* Accept the defaults for the inbound port rules for now; these rules will be configured later in the setup process.
+
+![Basics tab](../../images/azure_config-basics-co.png)
+
+Click **Next: Disks** to continue to the next tab.
+
+### 3. Configure disks
+
+Azure separates the main disk for the operating system and any data disks. On the **Disks** tab of the virtual machine configuration, you can configure the OS disk and attach data disks.
+
+For the Anbox Cloud Appliance, you should attach a separate data disk of at least 50 GB. To do so, click **Create and attach a new disk**. You can accept the default settings and change the disk size according to your requirements. For performance reasons, we recommend using 100 GB or more.
+
+![Create and attach a new disk](../../images/azure_config-disk.png)
+
+Click **Next: Networking** to continue to the next tab.
+
+### 4. Configure networking
+
+For networking, the Anbox Cloud Appliance requires the following changes to the default settings:
+
+* Configure the public IP with a basic SKU. The default standard SKU does not work.
+
+  To do so, create a public IP and select **Basic** under **SKU**.
```

Comment:
Can you file a task so we drop this once 1.15.2 goes out?

Reply:
Done.